src/lib.rs

```diff
@@ -15,25 +15,30 @@ use async_trait::async_trait;
 use prost::Message;
 use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
+use uprotocol_sdk::rpc::RpcServer;
```

Comment:
Consider moving all `uprotocol_sdk::foo` items under the larger `use uprotocol_sdk::{}` below.

Reply:
Make sense.